Documentation/monogame_faq.md

```diff
@@ -3,5 +3,135 @@ This page contains a list of frequently asked questions.
 ### What software do I need to start?
 
 Depending on the platform you wish to develop for the following thing are needed:
- - Android - You need Xamarin.Android: http://android.xamarin.com/, it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
- - iOS - You need Xamarin.iOS: http://ios.xamarin.com/, it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+ - Android - You need Xamarin.Android: [http://android.xamarin.com/], it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+ - iOS - You need Xamarin.iOS: [http://ios.xamarin.com/], it works on Windows and Mac and it can be used in combination with either Visual Studio or Xamarin Studio.
+
+
+### Where do I start? 
+
+The quickest way to start is to download CartBlanche's samples: [https://github.com/CartBlanche/MonoGame-Samples https://github.com/CartBlanche/MonoGame-Samples]
```

Comment:
I think we are moving to https://github.com/MonoGame/MonoGame.Samples
The CartBlanche ones are very old and not up to date.